actor-testkit-typed/src/main/scala/org/apache/pekko/actor/testkit/typed/javadsl/TestKitJUnit5Extension.scala

```diff
@@ -29,8 +29,8 @@ final class TestKitJUnit5Extension() extends AfterAllCallback with BeforeTestExe
    * Get a reference to the field annotated with @JUnit5Testkit  [[JUnit5TestKit]]
    */
   override def beforeTestExecution(context: ExtensionContext): Unit = {
-
-    context.getTestInstance.ifPresent((instance: AnyRef) => {
+    val testInstance: Option[AnyRef] = Option.when(context.getTestInstance.isPresent)(context.getTestInstance.get())
```

Comment:
I think Option.when only appeared in Scala 2.13. We need to support Scala 2.12 still.

Reply:
i replaced Option.when with an if statement